### PR TITLE
Release/prod 03 27 2017

### PR DESCRIPTION
**Pull request by emilyseibert**
Prod Release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+CHANGELOG
+=========
+
+## Develop
+**Enhancements**
+* Create Access Keys in Config
```
Comment (dehli) [inline CHANGELOG.md]: Does this need to be under master now?

Comment (emilyseibert) [inline CHANGELOG.md]: nope thats still in dev

Comment (dehli) [inline CHANGELOG.md]: actually, how will this be in the CHANGELOG if it's nota also on this branch?

Comment (dehli) [inline CHANGELOG.md]: Oh, I see you committed to this branch separately to update the CHANGELOG.

Comment (emilyseibert) [inline CHANGELOG.md]: realized that it cmd-z 'd too many times and it added it in by accident. not a merge thing. i'm just bade at codez. updating

**Pull request merged**